frontend/public/locales/en/alerts.json

```
@@ -18,5 +18,19 @@
       "email-valid": "Enter an email address in the correct format, such as name@example.com",
       "preferred-language-required": "Select preferred language of communication"
     }
+  },
+  "confirm": {
+    "page-title": "Submission Confirmation",
+    "breadcrumbs": {
+      "subscribe": "Confirm to subscribe to CDCP email alerts"
+    },
+    "no-preferred-language-on-file": "No preferred language on file",
+    "confirmation-information-text": "A confirmation code, required to finist the registration process, will be sent to clientsname@domain.com. This step must be completed within teh next 48 hours or by immediately entering the confirmation code below.",
+    "confirmation-completed-text": "Upon completion, you will begin to receive email alerts when important information for your Canada Dental Care Plan application is available.",
+    "confirmation-selected-language": "Selected Alert Me language preference:",
```

```suggestion
    "confirmation-selected-language": "Selected language preference:",
```

(Same comment for French)

---

Ive added a todo comment in the text, when we can retrieve the email address from the new source, i will modify this section to use it correctly